### PR TITLE
Added Url to internal links

### PR DESCRIPTION
**Pull request by marcelmoosbrugger**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | -
| Related issues/PRs | https://github.com/massiveart/husky/pull/723
| License | MIT
| Documentation PR | -

#### What's in this PR?

In the internal links, as well as in the smart-content, the url was added to the itembox.

### CODE REVIEW

**Comment by marcelmoosbrugger** [comment]
@danrot This PR can be reviewed.

File: src/Sulu/Bundle/ContentBundle/Resources/public/js/components/internal-links/main.js [hunk 1]
```diff
@@ -236,7 +237,9 @@ define([], function() {
             return templates.contentItem(
                 item[this.options.idKey],
                 item.title,
-                this.getItemIcons(item)
+                item.url,
+                this.getItemIcons(item),
+                this.sandbox.util.cropMiddle
```
Comment (danrot) [inline src/Sulu/Bundle/ContentBundle/Resources/public/js/components/internal-links/main.js]: Do we win something by passing the function like this? Makes things more complicated... If we need it this way I am tempted to say that we have to check in the template function if a function was passed.

File: src/Sulu/Bundle/ContentBundle/Resources/public/js/components/smart-content/main.js [hunk 2]
```diff
@@ -630,11 +634,13 @@ define(['services/husky/util'], function(util) {
...
                         dataId: item[this.options.idKey],
                         value: item[this.options.titleKey],
+                        description: item[this.options.descriptionKey] || null,
                         image: item[this.options.imageKey] || null,
                         webspace: this.options.webspace,
                         locale: this.options.locale,
                         num: (index + 1),
-                        icons: this.getItemIcons(item)
+                        icons: this.getItemIcons(item),
+                        cropper: this.sandbox.util.cropMiddle
```
Comment (danrot) [inline src/Sulu/Bundle/ContentBundle/Resources/public/js/components/smart-content/main.js]: Same as above

File: src/Sulu/Bundle/ContentBundle/Resources/public/js/components/internal-links/main.js (hunk 1 above)
Comment (danrot) [inline src/Sulu/Bundle/ContentBundle/Resources/public/js/components/internal-links/main.js]: Ah, different lengths, got it... Maybe we should introduce the check then. Same for the content I put below.

**Comment by danrot** [comment]
This is not working at all without a rebase...

**Comment by marcelmoosbrugger** [comment]
@danrot Your comments should be fixed now. I added a check for the cropping function in the templates.

**Comment by danrot** [comment]
Please rebase

**Pull request merged**